include/xeus-sqlite/xeus_sqlite_interpreter.hpp

```diff
@@ -18,38 +21,41 @@ namespace nl = nlohmann;
 
 namespace xeus_sqlite
 {
-    class XEUS_SQLITE_API interpreter : public xeus::xinterpreter
-    {
-    public:
-
-        interpreter() = default;
-
-        virtual ~interpreter() = default;
-
-    private:
-
-        void configure_impl() override;
-
-        nl::json execute_request_impl(int execution_counter,
-                                      const std::string& code,
-                                      bool silent,
-                                      bool store_history,
-                                      nl::json user_expressions,
-                                      bool allow_stdin) override;
-
-        nl::json complete_request_impl(const std::string& code,
-                                       int cursor_pos) override;
-
-        nl::json inspect_request_impl(const std::string& code,
-                                      int cursor_pos,
-                                      int detail_level) override;
-
-        nl::json is_complete_request_impl(const std::string& code) override;
-
-        nl::json kernel_info_request_impl() override;
-
-        void shutdown_request_impl() override;
-    };
+class XEUS_SQLITE_API interpreter : public xeus::xinterpreter
+{
+public:
+
+    interpreter() = default;
+
+    virtual ~interpreter() = default;
+
+    std::unique_ptr<SQLite::Database> db = nullptr;
+
```

Comment:
Is it okay to add members here?
Should I maybe create a new class?

Reply:
looks totally fine to me. 
You might want ot make it private though. Also our usual pattern is `m_db` (so member vars have a `m_`.

Reply:
Alright!